.github/workflows/deploy-dev.yml

```
@@ -0,0 +1,101 @@
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::049759450795:role/ShortLinkDevECRPushPolicy
+          aws-region: ap-northeast-2
+
+      - name: Get EC2 instance IP from Terraform
+        id: get-ip
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:7:26: Double quote to prevent globbing and word splitting

(shellcheck)